docs/admin/installation/servers.rst

```diff
@@ -61,12 +61,6 @@ display a message on boot that shows which key should be pressed to
 enter the boot menu. Once you've entered the boot menu, select the
 installation media (USB or CD) and press Enter to boot it.
 
-On newer hardware, such as the NUC12s, you may need to use a newer Linux
-kernel than the one that ships by default in **Ubuntu Server 20.04.6** in
-order to have more up-to-date hardware drivers. To use a newer Linux kernel,
-select **Boot and Install with the HWE Kernel** in the initial OS boot menu
-that appears prior to booting the Ubuntu image.
```

Comment:
There's a part of me that wonders if it's worth keeping this in, and just updating the language for 24.04.2. Although this LTS has a much newer kernel than Focal, some of the newer hardware coming out might require even newer ones.

Reply:
Yes, this was meant to prompt this discussion.  :-)  I've retained and revised this paragraph in 2620a88e6fd3c1e450669fa88e5ce2c77de1a28c.